app/views/replies/_reply.html.erb

```diff
@@ -42,9 +52,11 @@
 
       <% if user_signed_in? %>
         <% if reply.user.id == current_user.id %>
-          <%= link_to "Delete", {:action => 'destroy', :controller => "replies", :id => @reply.id, :new_thread_id => @new_thread.id}, :method => "delete", data: { confirm: 'Are you sure?' }, :class => 'reply-link edit-post'%>
+          <a id="edit-reply-btn-<%= reply.id %>" class="btn btn-default" reply-id="<%= reply.id %>" onClick="toggleEdit(this)"><i class="fa fa-pencil"></i></a>
```

Comment:
Actually I haven't pushed this code. I was trying this in my local and since it wasn't correct I didn't update it. Perhaps I should update it so that you can make a new review and make things clearer.